app/src/renderer/components/network/PageBlock.vue

```diff
@@ -81,21 +81,37 @@ export default {
       ]
     },
     blockTitle({ num, block } = this) {
+      if (!block.block) return `n/a`
       return `#` + num.prettyInt(block.block.header.height)
     },
     blockTime({ moment, block } = this) {
+      if (!block.block) return `n/a`
       return moment(block.block.header.time).format(`MMM Do YYYY, HH:mm:ss`)
     }
   },
   watch: {
-    $route: `getBlock`
+    "$route.params.height": function() {
```

Comment:
you can watch the `lastHeader` instead here

Reply:
No, this is watching if the route changes, not the chain height